plugins/k8saudit-gke/hack/falco.yaml

```diff
@@ -0,0 +1,77 @@
+base_syscalls:
+  custom_set: []
+  repair: false
+buffered_outputs: false
+file_output:
+  enabled: false
+grpc:
+  bind_address: unix:///run/falco/falco.sock
+  enabled: false
+  threadiness: 0
+grpc_output:
+  enabled: false
+http_output:
+  ca_bundle: ""
+  ca_cert: ""
+  ca_path: /etc/ssl/certs
+  enabled: false
+  insecure: false
+  url: "http://127.0.0.1:2801/"
+  user_agent: falcosecurity/falco
+json_include_output_property: true
+json_include_tags_property: true
+json_output: true
+libs_logger:
+  enabled: false
+load_plugins: [ k8saudit-gke, json ]
+log_level: info
+log_stderr: true
+log_syslog: true
+metadata_download:
+  chunk_wait_us: 1000
+  max_mb: 100
+  watch_freq_sec: 1
+metrics:
+  convert_memory_to_mb: true
+  enabled: false
+  include_empty_values: false
+  interval: 5m
+  kernel_event_counters_enabled: true
+  libbpf_stats_enabled: true
+  output_rule: true
+  resource_utilization_enabled: true
+modern_bpf:
+  cpus_for_each_syscall_buffer: 2
+output_timeout: 2000
+plugins:
+- init_config:
+    project_id: "te000-589aa8c5b4314ba934a6"
+    fetch_cluster_metadata: false
+    max_event_size: 8000000
+  library_path: libk8saudit-gke.so
+  name: k8saudit-gke
+  # open_params: "file://test/pods_create.json"
+  open_params: "falco-gke-audit-sub"
```

Comment:
are the `project_id` and `open_params` not sensitive in that example?

Reply:
`open_params` is the PubSub subscription id (you can define your own id/name when creating the subscription. It is the same id/name used in the readme. Without a (service)account with permissions to the subscription you can't access it.

`project_id` requires Google IAM, so it is not sensitive in that sense. But yeah, no point in having my google project id in it, so I will change this to `your-gcp-project-id`, same as in the readme 👍